File: src/main/java/com/gokhankorkmaz/loanapplicationsystem/business/dtos/responses/CustomerResponse.java
package com.gokhankorkmaz.loanapplicationsystem.business.dtos.responses;

import lombok.*;

import java.sql.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Builder
@EqualsAndHashCode
public class CustomerResponse {
    private int id;
    private String name;
    private String surname;
    private String identityNumber;
    private String phoneNumber;
    private Date birthdate;
    private double monthlyIncome;
    private int creditRating;
    private double assurance;

}

File: angular/loan-app-system/src/app/models/Customer/CustomerResponse.ts
import { Credit } from "../Credit/Credit";

export class CustomerResponse {
    id: number;
    identityNumber: string;
    name: string;
    surname: string;
    phoneNumber: string;
    birthdate: Date;
    monthlyIncome: number;
    creditRating: number;
    assurance: number;
}